corehq/apps/users/models.py

```diff
@@ -2342,6 +2342,11 @@ def get_device(self, device_id):
             if device.device_id == device_id:
                 return device
 
+    def get_devices_fcm_token(self):
```

Comment:
nit: `get_devices_fcm_tokens` or `get_fcm_tokens` or `get_fcm_tokens_from_devices`?

Reply:
lol had a hard time naming this. Thanks for catching this up. Will use `get_devices_fcm_tokens`

Reply:
Resolved in f81ef437a0c8d3971ffe3217c42ce98db6b35e93